cfpb/cfgov-refresh
# Update logos and text references to BCFP

Only thing done so far is the main header, behind a new `BCFP_LOGO` feature flag.

## Additions

- BCFP logos and homepage hero images
- `BCFP_LOGO` feature flag

## Removals

- Unused `home-hero_med.jpg` image
- Unused Ask CFPB `_search.html` template
- Unused Find a Housing Counselor PDF view (this is not what is currently used to generate our PDFs)
- Unused `base_print.html` legacy template

## Changes

- If `BCFP_LOGO` flag is on:
  - Display BCFP logo in main header (including on Spanish site)
  - Update text of homepage hero
  - Update page titles
  - Display BCFP logo on print view in Find a Housing Counselor and updates the name in the text on that view
  - Display BCFP logo as default Open Graph image (sharing to Facebook and Twitter) when a page-specific one is not given

## Testing

1. Pull branch
1. `gulp images`
1. Turn the `BCFP_LOGO` flag on
1. Load up http://localhost:8000/ and see the new logo, hero text and graphic
1. View it in responsive design mode and make sure all widths look good
1. Go to http://localhost:8000/es/ and see the new logo
1. Go to http://localhost:8000/find-a-housing-counselor/ and Print Preview and see the new logo
1. Back on the homepage, View Source and see that it's specifying the new `bfcp-logo` Open Graph images (the `<meta property="og:image" …>` tags

## Todos

- Replace other instances listed in GHE#2872
- Clean up Feature Flag once it's been flipped on in production

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
